tests/conftest.py

```diff
@@ -91,12 +91,12 @@ def chainer_version(request):
     return request.param
 
 
-@pytest.fixture(scope='module', params=['1.4.1', '1.5.0', '1.6.0', '1.7.0', '1.8.0'])
+@pytest.fixture(scope='module', params=['1.8.0'])
```

Comment:
Should we use the DEFAULT_VERSION constant here to minimize duplication?

Reply:
Ah there's no duplicate here, for now each framework will only have one latest version always.

But seems get a LATEST_VERSION constant would be more clear to readers. I will apply that. Thanks!

Reply:
I was actually thinking about just use these - https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/mxnet/defaults.py#L15